exercises/queen-attack/queen_attack.py

```diff
@@ -1,6 +1,6 @@
-def board():
+def board(posW, posB):
```

Comment:
In Python `snake_case` is favored over `CamelCase`.
I suggest to rename the params to meet the convention. More info - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

And I am not quite sure what `W` and `B` means

Reply:
yes, changed to `snake_case`
also the `W` and `B` are for `white` and `black` respectively.
I modified the PR to better reflect that.